src/open_inwoner/accounts/models.py

```diff
@@ -278,11 +278,11 @@ class Meta:
                 | Q(login_type=LoginTypeChoices.eherkenning),
                 name="check_kvk_or_rsin_only_set_when_login_eherkenning",
             ),
-            CheckConstraint(
-                check=((Q(kvk="") & ~Q(rsin="")) | (~Q(kvk="") & Q(rsin="")))
-                | ~Q(login_type=LoginTypeChoices.eherkenning),
-                name="check_kvk_or_rsin_exclusive_when_login_eherkenning",
-            ),
+            # CheckConstraint(
```

Comment:
I left this one out for now, because our current implementation uses both KVK and RSIN (OpenZaak requires RSIN and eSuite KVK)